src/main/scala/com/yotpo/metorikku/metric/stepActions/Sql.scala

```diff
@@ -21,13 +22,28 @@ case class Sql(query: String, dataFrameName: String, showPreviewLines: Int,
       case _ =>
     }
 
-    val newDf = sparkSession.sqlContext.sql(query)
+    val newDf = checkpointDataframe(dataFrameName, sparkSession.sqlContext.sql(query), checkpoint)
```

Comment:
val newDf = sparkSession.sqlContext.sql(query)
val newDfCp = checkpoint match {
                                case Some(true) => checkpointDataframe(dataFrameName, newDf, checkpoint)
                                case _ => newDf
                                }
                    
WDYT?

Reply:
I need to verify `getCheckpointDir` has value too, so I do everything in the same function